blocks/library/image/editor.scss

```diff
@@ -92,3 +96,30 @@
 		margin-right: auto;
 	}
 }
+
+.edit-post-sidebar .blocks-image-dimensions {
+	margin-bottom: 1em;
+
+	.blocks-image-dimensions__row {
+		display: flex;
```

Comment:
Changes to flex layout are always risky in IE11, but in my IE tests things are looking fine :)